components/dropdown/dropdown-content-mixin.js

```diff
@@ -893,21 +895,21 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 	_getDropdownStyling() {
 		const widthStyle = {
-			maxWidth: this.maxWidth ? `${this.maxWidth}px` : '',
-			minWidth: this.minWidth ? `${this.minWidth}px` : '',
+			maxWidth: this.maxWidth ? `${this.maxWidth}px` : null,
+			minWidth: this.minWidth ? `${this.minWidth}px` : null,
 			/* add 2 to content width since scrollWidth does not include border */
-			width: this._width ? `${this._width + 20}px` : ''
+			width: this._width ? `${this._width + 20}px` : null
 		};
 
 		const contentWidthStyle = {
-			minWidth: this.minWidth ? `${this.minWidth}px` : '',
+			minWidth: this.minWidth ? `${this.minWidth}px` : null,
 			/* set width of content in addition to width container so header and footer borders are full width */
-			width: this._width ? `${this._width + 18}px` : '',
+			width: this._width ? `${this._width + 18}px` : null,
```

Comment:
I noticed that these being empty string was causing an invalid property value caution symbol in Firefox. Let me know if there was a reason they were empty string instead of null (which causes `styleMap` to just not include them).

<img width="262" alt="Screenshot 2025-05-07 at 11 17 17 AM" src="https://github.com/user-attachments/assets/2a956ad0-9fcd-4116-87a7-cf99dd7af47b" />